components/feature/containers/src/main/java/mozilla/components/feature/containers/ContainerStorage.kt

```diff
@@ -20,7 +20,7 @@ import java.util.UUID
 /**
  * A storage implementation for organizing containers (contextual identities).
  */
-internal class ContainerStorage(context: Context) {
+class ContainerStorage(context: Context) {
```

Comment:
We talked about setting the visibility of `ContainerStorage` to `internal` in https://github.com/mozilla-mobile/android-components/pull/7623#pullrequestreview-446166064. 

> Can we set the visibility of ContainerStorage to internal? This way apps cannot access it and I assume only the middleware will need it? Or do we want to pass it to the middleware, which makes sense as a dependency structure, but I'd love to avoid the app ever doing the wrong thing by talking to the storage directly. :)

However, I didn't find another way to keep it `internal` while also mocking `ContainerStorage` in our `ContainerMiddlewareTest` tests.

Reply:
Fixed.